hill-day/index.html

```diff
@@ -199,12 +199,7 @@ <h3>FAQ</h3>
 				</article>
 				<article id="is-it-too-late-to-sign-up">
 					<div class="question">Is it too late to sign up?</div>
-					<div class="answer">No! If you are from: Alabama, Alaska, Arizona, Arkansas,
-						Hawaii, Idaho, Iowa, Kentucky, Louisiana, Mississippi, Nebraska, North
-						Dakota, South Carolina, South Dakota, West Virginia, or Wyoming we would
-						love to have you at the hill day! Just <a
-							href="https://docs.google.com/forms/d/e/1FAIpQLSdgRiCPmOlQnVgc6mjQ5S9q4a7AVTvCE1Qgw-FYT5vxIdp-vA/viewform"
-							target="_blank">sign up here</a>!</div>
+					<div class="answer">Signups for Hill Day 2025 have closed, but you can continue to call, email, and write to your representatives to advocate for more tuberculosis funding.</div>
```

Comment:
could you link to the templates like "You can find templates etc. on our templates page" and the whole thing is a link